Makefile

```
@@ -53,8 +53,9 @@ check-secrets:
 
 .PHONY: install-spire
 install-spire:
-	@read -p "Would you like to install Spire? [y/N] " response; \
-	if [ "$$response" = "y" ]; then \
+	$(eval IS=$(shell grep -A3 'spire:' global.yaml | grep enabled: | awk '{print $$2}'))
```

Rather than running an `eval` statement, let's use this instead
```suggestion
IS=$(cat global.yaml| grep -A3 'spire:'| grep enabled: | awk '{print $2}')
```

---

That wont work.  because make executes commands line to line as independent scripts.  this was the only way i could find to create envars that can be used throughout targets.

---

refering to `eval` and `shell`